.buildkite/filebeat/filebeat-pipeline.yml

```diff
@@ -189,44 +195,50 @@ steps:
           - "filebeat/build/*.json"
         notify:
           - github_commit_status:
-              context: "filebeat: Extended Ubuntu ARM Unit Tests"
+              context: "filebeat: Linux arm64 Unit Tests"
+
+  - group: "Filebeat Extended Tests"
+    key: "filebeat-extended-tests"
+    if: build.env("BUILDKITE_PULL_REQUEST") == "false" || build.env("GITHUB_PR_LABELS") =~ /.*(macOS).*/
 
-  - group: "Extended Windows Tests"
-    key: "filebeat-extended-win-tests"
-    if: build.env("BUILDKITE_PULL_REQUEST") == "false" || build.env("GITHUB_PR_LABELS") =~ /.*[Ww]indows.*/
     steps:
-      - label: ":windows: Windows 2019 Unit Tests"
-        key: "windows-extended-2019"
+      - label: ":mac: Filebeat macOS x86_64 Unit Tests"
+        key: "macos-unit-tests-extended"
+        if: build.env("BUILDKITE_PULL_REQUEST") == "false" || build.env("GITHUB_PR_LABELS") =~ /.*macOS.*/
```

Comment:
we shouldn't be running macos tests with merge commits, see https://github.com/elastic/beats/pull/39533/files
therefore we should remove `build.env("BUILDKITE_PULL_REQUEST") == "false"`

Reply:
7.17 changes are not merged yet: https://github.com/elastic/beats/pull/39553

Reply:
fixed